common/scala/src/main/resources/application.conf

```diff
@@ -12,3 +12,22 @@ akka.http {
         max-open-requests = 128
     }
 }
+
+akka.tracing {
+    #zipkin configuration
+    host = "localhost"
+    host = ${?TRACING_HOST} # Override default HOST by enviornment variable
+    port = 9410 # default
+    port = ${?TRACING_PORT} # Override default PORT by enviornment variable
+    sample-rate = 100 # by default trace every 100th request
```

Comment:
Thanks Adrian for your comment. But I think sample rate in akka-tracing lib is being treated as `int` [(https://github.com/levkhomich/akka-tracing/blob/master/core/src/main/scala/com/github/levkhomich/akka/tracing/TracingExtension.scala#L43)](https://github.com/levkhomich/akka-tracing/blob/master/core/src/main/scala/com/github/levkhomich/akka/tracing/TracingExtension.scala#L43) and does a mod on request counter. 
([https://github.com/levkhomich/akka-tracing/blob/master/core/src/main/scala/com/github/levkhomich/akka/tracing/TracingExtension.scala#L324](https://github.com/levkhomich/akka-tracing/blob/master/core/src/main/scala/com/github/levkhomich/akka/tracing/TracingExtension.scala#L324))

So a value between 0 - 1.0 would switch off the tracing.
I have kept a default of 100 which makes default tracing as 1%.